docs/mobile-apps/features/gestures.md

```diff
@@ -10,6 +10,31 @@ Building a great user experience is more than just design. Equally important is
 - [Sauce Labs Mobile App Gestures | GitHub](https://github.com/saucelabs/sample-app-mobile/#gestures)
 - [How To Do Multi-Touch Gestures in Live Testing](https://saucelabs.com/blog/how-to-do-multi-touch-gestures-in-live-testing)
 
-## What You'll Need
+### What You'll Need
 
 - A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- 
+
+
+### What gestures are supported on Real Devices?
+
+## On iOS we are supporting the following gestures:
+
+- **Pinch to Zoom**: Exclusive to iOS, activate using your trackpad or holding the ALT(Windows)/Option(MacOS) key [during a Live Testing session.](https://saucelabs.com/_next/image?url=https%3A%2F%2Fimages.ctfassets.net%2Fvrc8wif0t20g%2F72k6sPCuyuxDj0XUECBKs3%2Fa8df5afe7be23b6f1017e793b037e34c%2Fpinch_zoom.gif&w=3840&q=75)
+- **Two-Finger Scroll**: Captures both rapid and gradual scrolling actions.
+- **Mouse/Trackpad Journey**: Replicates intricate touch actions such as:
+  - Drag and Drop
+  - Sliders
+  - Complex screen drawings
```

Comment:
Should we call out more explicitly that you 
- can draw circles
- open the home menu from swiping from the bottom (which our competitors don't support)